content/en/tracing/service_catalog/integrations.md

```diff
@@ -4,15 +4,53 @@ kind: documentation
 further_reading:
 - link: "/tracing/service_catalog/service_definition_api/"
   tag: "Documentation"
-  text: "Service Definition API"
+  text: "Learn about the Service Definition API"
+- link: "/integrations/opsgenie/"
+  tag: "Documentation"
+  text: "Learn about the OpsGenie integration"
+- link: "/integrations/pagerduty/"
+  tag: "Documentation"
+  text: "Learn about the PagerDuty integration"
 ---
 
+## Overview
+
+When you configure a service account for a [Datadog integration][10], you can incorporate metadata from your integrations into service definitions in the [Service Catalog][9]. You can also utilize autocomplete and validation features when editing a service definition in an [integrated development environment (IDE)](#ide-integrations).
+
+## OpsGenie integration
+
+You can add OpsGenie metadata to a service so that the [Service Catalog][11] displays and links to information such as who is on-call for the service.
+
+### Setup
+
+1. If you have not already done so, set up the [Datadog OpsGenie integration][12].
+2. Get your OpsGenie API access key as described in their [API Key Management][13] documentation. This API key requires **configuration access** and **read** access rights.
+3. Add an account in the **Accounts** section on the [integration tile][14], paste your OpsGenie API access key, and select the region for your OpsGenie account. 
+   
+   {{< img src="tracing/service_catalog/create_account.png" alt="The Create New Account workflow in the OpsGenie integration tile" style="width:80%;" >}}
+
+4. Update the service definition with the OpsGenie information to link your OpsGenie service with your Datadog service. For example, pass in the following `integrations` configuration lines within the full [service definition][5]:
+
+   ```yaml
+   "integrations": {
+     "opsgenie": {
+           "service-url": "https://www.opsgenies.com/service/123e4567-e89b-12d3-a456-426614174000",
+           "region": "US"
+     }
+   }
+   ```
+
+Once you've completed these steps, an **On Call** information box appears in the **Ownership** tab of a service in the Service Catalog.
+
+{{< img src="tracing/service_catalog/oncall_information.png" alt="On Call information box displaying information from OpsGenie in the Service Catalog" style="width:85%;" >}}
+
 ## PagerDuty integration
-You can add PagerDuty metadata to a service so that Service Catalog displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.
+
+You can add PagerDuty metadata to a service so that the [Service Catalog][11] displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.
```

Comment:
```suggestion
You can add PagerDuty metadata to a service so that the Service Catalog displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.
```